src/components/organisms/pupil/OakPupilJourneyHeader/OakPupilJourneyHeader.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+
+import { OakFlex, OakHeading, OakIconProps } from "../../../atoms";
+import {
+  OakBulletList,
+  OakBulletListProps,
+  OakHandDrawnCardWithIcon,
+} from "../../../molecules";
+
+export type OakPupilJourneyHeaderProps = {
+  title: string;
+  iconName: OakIconProps["iconName"];
+  alt?: OakIconProps["alt"];
+  iconBackground?: "primary" | "secondary";
+} & OakBulletListProps;
+
+/**
+ * This component is the header for the pupil journey;
+ *
+ * the icon, title and list of items are passed as props and change change depending on which page it is called
+ *
+ *
+ */
+export const OakPupilJourneyHeader = ({
+  iconBackground = "primary",
+  ...props
+}: OakPupilJourneyHeaderProps) => {
+  return (
+    <OakFlex $flexDirection={"row"} $gap={"space-between-m"} {...props}>
+      <OakHandDrawnCardWithIcon
```

Comment:
i didnt use the OakSubjectIcon component as the dimentions were different; and the breakpoints set didnt match the use in this title component

Reply:
Fair enough, but woulkd it work now that Mike has added breakpoints. No worries if not but I guess you need to implement his changes in anycase